frontend/src/ApplicationModal.tsx

```diff
@@ -74,7 +77,7 @@ const ApplicationModal: React.FC<ApplicationModalProps> = ({ isOpen, onClose, ap
         );
         setTimeout(() => {
           window.location.reload();
-        }, 700);
+        }, 800);
```

Comment:
Why are we doing this? Could we not make another API call to fetch the data again or something?

Reply:
ya i tried that but i was not working i will check again

Reply:
If you haven't been able to figure it out, it's okay. Let's just move on with this.